packages/editor/src/setupTests.ts

```diff
@@ -1,6 +1,10 @@
+import 'common/lib/polyfills';
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
+// tslint:disable-next-line:no-var-requires
+(window as any).require = require('../public/external/monaco-editor-0-14-3/vs/loader');
```

Comment:
There's a constant for `monaco-editor-0-14-3`.
`\packages\common\src\package-versions.ts`
`HYPHENATED_PACKAGE_VERSIONS`